packages/mongo-nav/src/org-nav/OrgNav.tsx

```diff
@@ -307,6 +307,7 @@ function OrgNav({
                 usePortal={false}
               >
                 <MenuItem
+                  as={orgNav.accessManager ? 'a' : 'button'}
```

Comment:
I'm surprised this typechecks since from TS's point of view we might be giving an "href" to a "button". is there a chance that `MenuItem`'s typing is not catching error cases?

Reply:
Definitely possible... 🤔 But 
```
<MenuItem target="xx" />;
<MenuItem target="xx" as="button" />;
```

Both of these fail as expected

Reply:
Maybe TS is smart enough to understand we're checking that the same value used for href is truthy here?